docs/providers/hetzner.md

```diff
@@ -0,0 +1,233 @@
+# Hetzner Cloud Provider
```

Comment:
Please include these instructions as part of the main docs.md and not a separate file like this.

P.S.: You would need to raise another PR to the docs repo with similar changes.